### PR TITLE
dms: init at 1.6.0

### PR DESCRIPTION
**Pull request by claes**
## Description of changes

dms is a UPnP DLNA Digital Media Server, written in Go. 

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/dm/dms/package.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "dms";
+  version = "1.6.0";
+  src = fetchFromGitHub {
+    owner = "anacrolix";
+    repo = pname;
```
Comment (Aleksanaa) [inline pkgs/by-name/dm/dms/package.nix:12]:
```suggestion
    repo = "dms";
```

File: pkgs/by-name/dm/dms/package.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    rev = "v${version}";
+    sha256 = "sha256-QwRLNCXDu/dKh2o17AyASlVQPIEOX6e4kTINa2ZzZkU=";
```
Comment (Aleksanaa) [inline pkgs/by-name/dm/dms/package.nix:14]:
```suggestion
    hash = "sha256-QwRLNCXDu/dKh2o17AyASlVQPIEOX6e4kTINa2ZzZkU=";
```

File: pkgs/by-name/dm/dms/package.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "dms";
+  version = "1.6.0";
```
Comment (Aleksanaa) [inline pkgs/by-name/dm/dms/package.nix:9]:
```suggestion
  version = "1.6.0";

```

File: pkgs/by-name/dm/dms/package.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromGitHub,
+}:
```
Comment (Aleksanaa) [inline pkgs/by-name/dm/dms/package.nix:6]:
```suggestion
}:

```

Comment (claes) [reply pkgs/by-name/dm/dms/package.nix:6]: Updated

File: pkgs/by-name/dm/dms/package.nix (hunk 3 above)
Comment (claes) [reply pkgs/by-name/dm/dms/package.nix:10]: Updated

File: pkgs/by-name/dm/dms/package.nix (hunk 1 above)
Comment (claes) [reply pkgs/by-name/dm/dms/package.nix]: Updated

File: pkgs/by-name/dm/dms/package.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+buildGoModule rec {
+  pname = "dms";
+  version = "1.6.0";
+  src = fetchFromGitHub {
+    owner = "anacrolix";
+    repo = pname;
+    rev = "v${version}";
```
[the remark this answers is not in the document]
Comment (claes) [reply pkgs/by-name/dm/dms/package.nix]: Updated

File: pkgs/by-name/dm/dms/package.nix [hunk 6]
```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  stdenv,
```
Comment (Aleksanaa) [inline pkgs/by-name/dm/dms/package.nix:3]:
```suggestion
```

Not used

Comment (claes) [reply pkgs/by-name/dm/dms/package.nix]: Updated

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**